net/unbound/files/unbound.init

```diff
@@ -1,20 +1,718 @@
 #!/bin/sh /etc/rc.common
-# Copyright (C) 2016 Michael Hanselmann
```

Comment:
Sorry, that wasn't my intent. I started from scratch working within my router to assemble the conf file only (not launch binary). I used the CC 15.05.1 booked binary to manually launch on my conf file. Then added the actual launcher last. The provided init would often fail **restart** because Ubound would still be writing to syslog, while the new Unbound instance failed to get access to port 53. File copy back, not managed/merged in git. Anyway GPL2 flat okay?